### PR TITLE
Move multi-condition segments to Premium [MAILPOET-3929]

### PR DESCRIPTION
**Pull request by wxa**
[MAILPOET-3929](https://mailpoet.atlassian.net/browse/MAILPOET-3929)

### CODE REVIEW

File: assets/js/src/segments/dynamic/form.tsx [hunk 1]
```diff
@@ -150,21 +156,16 @@ export const Form: React.FunctionComponent<Props> = ({
...
-              const filters = segment.filters;
-              filters.push({
-                segmentType: SegmentTypes.WordPressRole,
-                action: SubscriberActionTypes.WORDPRESS_ROLE,
-              });
-              updateSegment({
-                filters,
-              });
-            }}
+            onClick={(e): void => addConditionAction(e, segment, updateSegment)}
```
Comment (pavel-mailpoet) [inline assets/js/src/segments/dynamic/form.tsx:159]: Does the function `addConditionAction` really need the Event? Could we call the `preventDefault` function here and then not pass the event further?

**Review by pavel-mailpoet — COMMENTED** [review]
Good job Alex, just a small question.

File: assets/js/src/segments/dynamic/form.tsx (hunk 1 above)
Comment (wxa) [reply assets/js/src/segments/dynamic/form.tsx]: You're right, it's not needed in this callback, I've removed it.

**Review by pavel-mailpoet — DISMISSED** [review]

**Review by veljkho — CHANGES_REQUESTED** [review]
Everything seems good except one thing:
I am able to make an advanced segment even with a warning "Your current MailPoet plan does not support advanced segments.".. I used **3 subscribers plan** so I'm getting a [warning about upgrade](https://d.pr/i/9E6fLO), and I am blocked to edit advanced segments nor to send newsletter to them, but I [am just able to make an advanced segment](https://d.pr/i/eC1gDy).

**Comment by wxa** [comment]
@veljkho Fixed, please update both branches.

**Review by veljkho — DISMISSED** [review]

**Review by veljkho — APPROVED** [review]

**Pull request merged**